### DISCUSSION TITLE
Remove autogenerated supports option in Tile Inspector

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#16594] [Feature requests] [answered]
It would be nice if we had the option to remove supports generated by the game engine automatically like we have the option to remove park fences. Right now there are unavoidable glitches when removing ride supports over water and there are numerous guides and workarounds for how to reorder sprites or use scenery to hide supports.

**Comment by ZehMatt** [comment, marked as answer]
I believe this is the same request as https://github.com/OpenRCT2/OpenRCT2/issues/15346

**Reply by 733737** [reply]
that thread is specific to rct1 sv4 and says to not add it to tile inspector. I am requesting it to be added to tile inspector

**Reply by ZehMatt** [reply]
Its not specific to RCT1 and it could be added to the Tile Inspector if such flag is stored, the Tile Inspector is just a reflection of features the game already has and usually doesn't implement anything new.

**Reply by 733737** [reply]
I was suggesting adding a code function to check whether the engine is adding supports, which would require some form of reverse engineering, I assume

**Reply by duncanspumpkin** [reply]
You have just described 15346. The aim of that is to add a flag that indicates if the supports should draw or not.

**Reply by 733737** [reply]
Why don't you want players to have access to this feature?

**Reply by ZehMatt** [reply]
No one said we don't want this feature, we are saying this is already being requested and is precisely what you describe, "a way to disable drawing supports".

**Reply by 733737** [reply]
ok, should i close this request or should it be merged with the other one since they aren't identical?

**Comment by spacek531** [comment]
https://github.com/OpenRCT2/OpenRCT2/pull/15345

EDIT: whoops didn't see duncan's reply

**Reply by 733737** [reply]
sounds like the code is there, I would be very happy if you decided the feature could be added to 4.0 and I know others that would be happy also

**Reply by duncanspumpkin** [reply]
The code unfortunately is not there yet. I had a test idea but decided the approach wouldn't be the best. I'm still debating how it should be done.

**Reply by spacek531** [reply]
My code does flag each track element as having supports, but there has not been any verification to ensure it works properly with all track types aside from known issues. It also does not have tile inspector options yet.

**Reply by 733737** [reply]
I will close this duplicate request and wish you luck

**Comment by TheClaudeQC** [comment]
> It would be nice if we had the option to remove supports generated by the game engine automatically like we have the option to remove park fences. Right now there are unavoidable glitches when removing ride supports over water and there are numerous guides and workarounds for how to reorder sprites or use scenery to hide supports.

did you seek, plants and use tile inspector to make them underground, whiteout changing the order?
